### ISSUE TITLE
addon/azure_devops_v1 should be applied on level1 instead of level0

### ISSUE THREAD

**Issue by arne21a** [Azure/caf-terraform-landingzones#323]
tl;dr:
I think the docs of the addon are wrong and it should state 
```
rover -lz /tf/caf/caf_solution/add-ons/azure_devops_v1 \
  -tfstate azure_devops-contoso_demo.tfstate \
  -var-folder /tf/caf/caf_solution/add-ons/azure_devops_v1/scenario/200-contoso_demo \
  -parallelism 30 \
  -level level0 \ --> level1
  -env sandpit \
  -a apply
````

**my scenario/ how to reporduce this**
I just rolled out the caf launchpad with a modified [200 scenario](https://github.com/Azure/caf-terraform-landingzones/tree/main/caf_launchpad/scenario/200).  
When adding the [azure_devops_v1 addon](https://github.com/Azure/caf-terraform-landingzones/tree/main/caf_solution/add-ons/azure_devops_v1) i noticed that the documented command does not work as expected.
```
rover -lz /tf/caf/caf_solution/add-ons/azure_devops_v1 \
  -tfstate azure_devops-contoso_demo.tfstate \
  -var-folder /tf/caf/caf_solution/add-ons/azure_devops_v1/scenario/200-contoso_demo \
  -parallelism 30 \
  -level level0 \
  -env sandpit \
  -a apply
````

upon closer inspection I figured that this addon has to be applied with ```-level level1``` flag. Otherwise the [landingzone.tfvars](https://github.com/Azure/caf-terraform-landingzones/blob/main/caf_solution/add-ons/azure_devops_v1/scenario/200-contoso_demo/landingzone.tfvars) does not work out.

The ```azure_devops-contoso_demo.tfstate``` is expected on level1
```
landingzone = {
  backend_type        = "azurerm"
  global_settings_key = "launchpad"
  level               = "level1"
  key                 = "azure_devops-contoso_demo.tfstate"
...
}
```
the caf_launchpad.tfstate (level0) state is expected as "lower" state. When we roll this out with ```-level level0``` there can not be a lower state. So we get this error:
```
@calling terraform_plan -- azurerm
╷
│ Error: Error loading state error
│ 
│   with data.terraform_remote_state.remote["launchpad"],
│   on locals.remote_tfstates.tf line 19, in data "terraform_remote_state" "remote":
│   19:   backend = var.landingzone.backend_type
│ 
│ error loading the remote state: blobs.Client#Get: Invalid input: `accountName` cannot be an empty string.
╵
Terraform plan return code: 1
```

Can you please confirm that i did nothing wrong? In exchange i would provide a pullrequest to fix the docs, in case im right. ;)

**Expected behavior**

documented command contains ```-level level1``` flag 

**Configuration (please complete the following information):**
 - OS and version: macos with rover(sh) in docker rover-preview:1.1.3-2201.190325 
 - Version of the rover: rover-preview:1.1.3-2201.190325
 - Version of the landing zone: current main branch with contoso scenario 467f0fd

**Comment by arnaudlh** [comment]
You are correct @arne21a - we are reworking that part to automate further the pipelines creation, but in the meantime, yeah the docs are not correct.
